mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -5269,6 +5162,7 @@ impl<'a> From<&'a TakerOrder> for TakerOrderForRpc<'a> {
     }
 }
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
To not allocate it in heap

Reply:
No problem, we use this in other places too :)